### PR TITLE
https://issues.redhat.com/browse/ACM-4436 Adding an additional step in automation template for user

### PR DESCRIPTION
**Pull request by faaden123**
This is an addition of an extra step to help with the template that explains how to create an automation. @robdolares if you could review this change, that would be great! Thank you so much!

### CODE REVIEW

**Review by robdolares — APPROVED** [review]
/lgtm

File: clusters/cluster_lifecycle/ansible_config_cluster.adoc [hunk 1]
```diff
@@ -46,7 +46,8 @@ To specify the details about running the Ansible template while creating a templ
 
 . Enter a unique name for your template, which contains lowercase alphanumeric characters or a hyphen (-).
 
-. Select the credential that you want to use for the new template. To link an Ansible credential to an Ansible template, complete the following steps:
+. Select the credential that you want to use for the new template.
+. Once a credential has been selected, you can select an Ansible inventory to use for all the jobs. To link an Ansible credential to an Ansible template, complete the following steps:
```
Comment (swopebe) [inline clusters/cluster_lifecycle/ansible_config_cluster.adoc]: Here is where writing concisely, in active voice, sets tech writing apart. Please take some time to review the style manual here to see what is said about the word "Once" and avoiding past participles (has been).

Rewrite as such (but please do take a min to study the guide):

After you select a credential, ....

**Review by swopebe — CHANGES_REQUESTED** [review]

File: clusters/cluster_lifecycle/ansible_config_cluster.adoc (hunk 1 above)
Comment (faaden123) [reply clusters/cluster_lifecycle/ansible_config_cluster.adoc]: Got it, Thank you.

Comment (faaden123) [reply clusters/cluster_lifecycle/ansible_config_cluster.adoc]: @swopebe If you have a chance to review this again as I made the necessary changes! Thank you:)

**Review by swopebe — APPROVED** [review]
Looks good

**Pull request merged**